tests/spec/core/dfn-panel-spec.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
+import { flushIframes, makeRSDoc, makeStandardOps } from "../SpecHelper.js";
+
+describe("Core — dfnPanel", () => {
+  afterAll(flushIframes);
+
+  const body = `
+    <section>
+      <h2>top level heading</h2>
+      <p><dfn>many</dfn>, <dfn>one</dfn>, <dfn>zero</dfn> references.</p>
+      <p>[=many=] [=many=] [=one=]</p>
+      <section>
+        <h3>nested section heading</h3>
+        <p>[=many=] [=many=]</p>
+      </section>
+      <p>[= many =]</p>
+    </section>
+  `;
+  const ops = makeStandardOps(null, body);
+
+  describe("dfnPanel state", () => {
+    it("opens panel on dfn click", async () => {
+      const doc = await makeRSDoc(ops);
```

Comment:
Hmm... I wonder if we can just create one `doc` and then just use that to check all the states?

Reply:
Or is this because random order is fun? :::)

Reply:
I guess after each test, you could put the dfn box back into its initial state?

Reply:
Would've to close the popup before each test case though.... 
That should be easy though (came to my mind while writing this reply)

Reply:
Random order causing issues :sob:

Reply:
ok, let's go with this for now... hopefully the random issue is not indicative of actual issues.